commands/channel_test.go

```diff
@@ -2176,3 +2166,243 @@ func (s *MmctlUnitTestSuite) TestRenameChannelCmd() {
 		s.Require().Equal(printer.GetLines()[0], updatedChannel)
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestCreateChannelCmd() {
+	s.Run("should not create channel without display name", func() {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+
+		teamName := "teamName"
+		channelName := "channelName"
+		args := []string{teamName + ":" + channelName}
+
+		cmd.Flags().String("team", teamName, "Team Name")
+		cmd.Flags().String("name", channelName, "Channel Name")
+
+		err := createChannelCmdF(s.client, cmd, args)
+		s.Require().EqualError(err, fmt.Sprintf("display Name is required"))
```

Comment:
There is no need to use `fmt.Sprintf` here:

```suggestion
		s.Require().EqualError(err, "display Name is required")
```